packages/next-swc/crates/next-core/src/next_shared/transforms/next_dynamic.rs

```diff
@@ -65,12 +65,9 @@ impl CustomTransformer for NextJsDynamic {
             },
             self.is_server,
             self.is_server_components,
-            NextDynamicMode::Turbopack {
-                dynamic_transition_name: match self.mode {
-                    NextMode::Development => "next-client-chunks".to_string(),
-                    NextMode::Build => "next-dynamic".to_string(),
-                },
-            },
+            // [NOTE] We do not use `NextDynamicMode::Turbopack` yet, it isn't compatible with
+            // current loadable manifest, which causes hydration errors.
```

Comment:
Nit: Move this to the docs in `packages/next-swc/crates/next-transform-dynamic/src/lib.rs`